spec/requests/feedbacks_spec.rb

```diff
@@ -7,10 +7,16 @@
     let(:feedback) { Feedback.order(created_at: :asc).last }
     let(:feedback_params) { params[:feedback] }
     let(:provider) { create :provider }
-    let(:session_vars) { {} }
-    let(:address_lookup_page) { 'http://localhost:3000/providers/applications/fa580023-5b07-493d-bb64-49b6d97c2a97/address_lookup' }
+    let(:session_vars) do
+      {
+        'page_history' => [address_lookup_page, '/feedback']
+      }
+    end
```

Comment:
This will need to change because of the new way of handling page histories. You can use the `PageHistoryService` to set the history:

```
  let(:page_history_service) { PageHistoryService.new(page_history_id: session['page_history_id']) }
  before { page_history_service.write([address_lookup_page, '/feedback']) }
```

Reply:
Perfetct, better than my suggestion. You're right that using a stub is the best solution in a spec.